.rhdh/docker/Dockerfile

```diff
@@ -229,6 +229,11 @@ ENV CHOKIDAR_USEPOLLING='1' CHOKIDAR_INTERVAL='10000'
 # To avoid running scripts when using `npm pack` to install dynamic plugins
 ENV NPM_CONFIG_ignore-scripts='true'
 
+# Segment key for production
+#TODO
+ENV SEGMENT_WRITE_KEY=""
```

Comment:
Sure, we can do that.
I assume that you will be doing something like `sed 's/SEGMENT_WRITE_KEY=.*$/SEGMENT_WRITE_KEY=<prodkey>/'`. It would be nice to have this in place before we merge this PR. Can you please make the necessary changes to sync-midstream.sh script?

Reply:
Occurs to me the only value in having one public key here and the other public key in gitlab is to obfuscate the prod key, but it's public and plain text in the container sources, so why bother? 

On the other hand, if we put the prod key here as default, then we could toggle it off only for some builds downstream (eg. anything built from the main branch)